Objects and Functions.js

```diff
@@ -5,66 +5,60 @@
 
 // -        В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
 let car = {
-  colore: "чёрный",
-  power: "мощность двигателя", 
+    color: 'чёрный',
+    power: 'мощность двигателя',
 };
-car.colore = "зелёный";
+car.color = 'зелёный';
 console.log(car);
 
 // -         На склад принимают груши и яблоки, напишите функцию, которая возвращает результат сложения количества принятых груш и яблок
 
-
 function calcSum(apple, pear) {
-if (apple + pear > 1) {
-  return apple + pear;
-} else {
-  return console.log('такое возможно?');
-}
+    if (apple + pear > 1) {
+        return apple + pear;
+    } else {
+        return console.log('такое возможно?');
+    }
 }
 
-//-        В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в 
+//-        В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в
 // терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
 
-
-
 user = {
-name: "John",
+    name: 'John',
 };
 
-function chekName(key) {
-if (user.name === key) {
-return true;
-} else {
-  return false;
+function checkName(key) {
+    if (user.name === key) {
+        return true;
+    } else {
+        return false;
+    }
 }
-}
-
-let key = prompt("введите ваше имя", "");
 
+let key = prompt('введите ваше имя', '');
 
-if ( chekName(key) ) {
-console.log('Привет ' + key);
+if (checkName(key)) {
+    console.log('Привет ' + key);
 } else {
-console.log('нет такого имени');
+    console.log('нет такого имени');
 }
 
 //-        Напишите функцию вычисления типа аргумента и вывод типа в консоль
 
-function getArgType(arg){
-const type = typeof arg;
-console.log(type);
+function getArgType(arg) {
+    const type = typeof arg;
+    console.log(type);
 }
 
 //-        Напишите функцию, которая определяет является ли число простым или нет
 
 function showPrime(num, div = 2) {
```

Comment:
This function does not work properly. Try 9 and 15, they are composite numbers, your function returns that it is a prime number.